{{ cookiecutter.__project_name_kebab }}/ruff.toml

```
@@ -32,3 +32,13 @@ lines-between-types = 1
 
 [format]
 docstring-code-format = true
+
+
+[per-file-ignores]
```

When I run ruff, this tells me

```
warning: The top-level linter settings are deprecated in favour of their counterparts in the `lint` section. Please update the following options in `ruff.toml`:
  - 'per-file-ignores' -> 'lint.per-file-ignores'
```

```suggestion
[lint.per-file-ignores]
```